exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,9 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var sum = numberOfMentors + numberOfStudents;
```

Comment:
Hi as a new developer it is better to use **let** or **const** instead of using **var.** The fact about const and let is you can’t re-assign another value to the constant x, however let allows you to declare variables.